src/Exporter.php

```diff
@@ -36,6 +36,13 @@
 
 final readonly class Exporter
```

Comment:
in case https://github.com/sebastianbergmann/phpunit/pull/5774 is a viable option for dataproviders we would need to use the object exporters also in `shortenedRecursiveExport`

Reply:
Regardless of https://github.com/sebastianbergmann/phpunit/pull/5774, I do not think that `shortenedRecursiveExport()` or `shortenedExport()` should support custom object comparison.